e2e/utils.go

```diff
@@ -62,6 +66,14 @@ func defaultTLSConfig() tlsConfig {
 	}
 }
 
+func newTLSConfig(tempDir string) tlsConfig {
```

Comment:
These method names (newTLSConfig and newMTLSConfig) feel a little off to me because they're specific to CAPath, but the name doesn't imply that.